src/__tests__/testScreen.py

```diff
@@ -173,7 +182,7 @@ def assertEqualLines(self, testName, actualLines, expectedLines):
             self.assertEqual(
                 expectedLine,
                 actualLine,
-                'Lines did not match for test %s:\n\nExpected:%s\nActual:%s' % (
+                'Lines did not match for test %s:\n\nExpected:%s\nActual  :%s' % (
```

Comment:
add some spaces here to line up the expected and actual for easy visual diffing.